.circleci/config.yml

```diff
@@ -19,58 +20,49 @@ push_builds_only: &push_builds_only
 executors:
   amphtml-docker-small-executor:
     docker:
-      - image: cimg/base:2021.05
+      - image: cimg/base:stable
     resource_class: small
   amphtml-medium-executor:
-    machine:
-      image: ubuntu-2004:202010-01
+    docker:
+      - image: cimg/node:lts-browsers
     resource_class: medium
   amphtml-large-executor:
-    machine:
-      image: ubuntu-2004:202010-01
+    docker:
+      - image: cimg/node:lts-browsers
     resource_class: large
```

Comment:
nit: Now that these two have moved from machine to docker, would it make sense to rename them to `amphtml-docker-{medium|large}-executor` to be consistent with `amphtml-docker-small-executor` above? Or alternatively, drop `docker-` from line 20?

Reply:
Yep, consistency is key. Let's remove the `-docker` tag, it doesn't really add much information given that the executor itself is defined with the keyword "docker" inside it. Done.